### DISCUSSION TITLE
Why my re verification is being rejected?

### DISCUSSION THREAD

**Discussion by ckr0** [community/community#52899] [GitHub Education]
So i am getting rejection mails with reason that my name is not per documents( it is by the way.)also something about bio. i have written something but i dont know what github expects.

my whole personal details? like family phone number address girlfriend?

my personal preferences like food sex etc?

i dont know what you could write in bio.

please suggest

**Comment by samus-aran** [comment]
Hi @ckr0 👋 

> my whole personal details? like family phone number address girlfriend?
> 
> my personal preferences like food sex etc?

Heh, that's not necessary 😅 But certainly hobbies and interests 👍 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. Does your name or evidence needs translated also?

We would also suggest that when you update your profile information that you **log out and log back** into GitHub before re-applying. 👈
